addons/hr_attendance/static/src/js/attendance.js

```diff
@@ -61,12 +61,9 @@ var AttendanceSlider = Widget.extend({
         var employee = new data.DataSetSearch(self, 'hr.employee', self.session.user_context, [
             ['user_id', '=', self.session.uid]
         ]);
-        return employee.read_slice(['id', 'name', 'state', 'last_sign', 'attendance_access']).then(function (res) {
```

Comment:
@hmo-odoo attendance_access field has been removed from model, do you mean we should keep attendance_access field in model, attendance_access field just checks whether user having group group_hr_attendance, does it make sense to have a field for checking group ?